use_cases/fetcher.go

```diff
@@ -23,22 +25,47 @@ func NewFetcher(api api, storage writer) Fetcher {
 	return Fetcher{api, storage}
 }
 
+type PokemonsByID []models.Pokemon
```

Comment:
Why redeclare this? Should it be a good idea de create an entity or declare this in the entities?